app/client/views/home/home.html

```diff
@@ -128,23 +110,85 @@ <h1>sponsors</h1>
 				<div class="page-header">
 					<h1>faq</h1>
 				</div>
-				<div>
-					"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-					Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-					irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-					cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-				</div>
-				<div>
-					"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-					Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-					irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-					cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-				</div>
-				<div>
-					"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-					Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-					irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-					cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+				<div class="page-text text-center row">
+					<div class="col-lg-4 col-md-4 col-sm-6 col-xs-12">
+						<div class="question">
+							What is a hackathon?
+						</div>
+						<div class="answer">
+							Although many people associate the word hacking with illegal activities a hackathon is completely the opposite. A hackathon
+							is event where people can come together and share their ideas to create something fun while also learning and having
+							a great time.
```

Comment:
Typos:
"Although many people associate the word hacking with illegal activities **,**[HERE] a hackathon is completely the opposite. A hackathon is **an** [HERE] event where.."